fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -228,6 +228,8 @@ public class PropertyAnalyzer {
 
     public static final String PROPERTIES_STORAGE_VOLUME = "storage_volume";
 
+    public static final String PROPERTIES_UNIFIED_CTAS_ENGINE = "unified_ctas_engine";
```

Comment:
Is it sufficient to use the `engine`?

Reply:
Currently, CTAS only supports Hive and Iceberg Engine.